prefactor.cwl

```diff
@@ -15,62 +15,86 @@ inputs:
 outputs:
   losoto_h5:
     type: File
+    format: h5parm
+    label: "H5Parm solutions"
     outputSource: h5imp_cal/losoto_h5
 
   dTEC_1st:
     type: File
+    format: numpy
```

Comment:
Seems a big vague, numpy has lots of object types..

Reply:
I meant it's a `.npy` file, with something stored inside it (which is actually just one array, but I didn't mean to specify that here). Is there an allowed way to be vague here?

Reply:
We've actually abandoned raw numpy files in the upstream prefactor, in favor of the better HDF5-based format where the metadata is stored as well. So probably I should just remove `numpy` everywhere.

Reply:
For future reference, a good URL for `.npy` appears to be https://docs.scipy.org/doc/numpy-dev/neps/npy-format.html

Reply:
Should it then be `application/x-hdf5` ?

Reply:
No, numpy and hdf5 are different things.
* numpy files: I'll replace numpy by the reference you mentioned. 
* h5parm files: h5parm is a specification on top of HDF5. So yes, it is a `application/x-hdf5`, but the h5parm specification defines a layout in the HDF5. I think the link to the cookbook PDF is a pretty good description (although it could be a nicer link).

Reply:
Okay, got it.